pkg/freeze/containerd/pause_test.go

```diff
@@ -0,0 +1,144 @@
+package containerd
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/containerd/containerd"
+	"google.golang.org/grpc"
+	cri "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"knative.dev/container-freezer/pkg/daemon"
+)
+
+const (
+	freezeMethod = "freeze"
+	thawMethod   = "thaw"
+)
+
+var (
+	containers []*cri.Container
+	results    []string
+	method     string
+)
+
+var (
+	queueProxy     = cri.Container{Id: "queueproxy", Metadata: &cri.ContainerMetadata{Name: "queue-proxy"}}
+	userContainer  = cri.Container{Id: "usercontainer", Metadata: &cri.ContainerMetadata{Name: "user-container"}}
+	userContainer2 = cri.Container{Id: "usercontainer2", Metadata: &cri.ContainerMetadata{Name: "user-container2"}}
```

Comment:
how about a `Container(id, name string)` method similar to how we do things like https://github.com/knative/serving/blob/1f774190e8dfe840f582f2ae9d6f16e40661548c/pkg/testing/v1/revision.go#L224